### PR TITLE
Forward to CONTRIBUTING.md; simplify.

### PR DESCRIPTION
**Pull request by Axel-Naumann**

### CODE REVIEW

File: contribute/index.md [hunk 1]
```diff
@@ -7,31 +7,26 @@ toc: true
...
+file shipped with every binary and source distribution.
+The copyright transfer helps us with effectively defending the project in case of litigation.
 
-Presently, there are two ways in which you can contribute:
+There are two ways in which you can contribute:
 
 1. **Via the Users' contribution section in the Forum** <br>
    The Users' Forum has a [section](https://root-forum.cern.ch/c/my-root-app-and-ideas){:target="_blank"}
-   which describes how to submit contributions. It is the
+   which describes how to submit sizable feature contributions. It is the
```
Comment (eguiraud) [inline contribute/index.md:22]: Does my-root-app-and-ideas really describe "how to submit sizable feature contributions"?

It's more a space for ROOT-related advertisement than for _contribution_, isn't it?

File: contribute/index.md [hunk 2]
```diff
@@ -7,31 +7,26 @@ toc: true
...
 
 2. **Sending Patches** <br>
-   You can send us a patch or a pull request
-   with [Github](https://github.com/root-project/root){:target="_blank"}, provided that you
-   follow these two simple rules:
-
-    - Make sure you follow the [ROOT coding conventions]({{'/contribute/coding_conventions' | relative_url }}) in your code
-    - Make sure you provide a set of tests for your feature/bug fix
-    - Make sure your git commits can be added to ROOT's master branch with a "rebase and merge", i.e. by fast-forwarding the master branch. This can be easily done by git-rebasing your branch on ROOT's master branch. ROOT tries to avoid noisy merge commits as much as possible, so that the log remains mostly linear and easy to understand at a glance.
+   You can send us a pull request with [Github](https://github.com/root-project/root){:target="_blank"}.
```
Comment (eguiraud) [inline contribute/index.md:28]:
```suggestion
   You can create a pull request on [GitHub](https://github.com/root-project/root){:target="_blank"}.
```

**Review by eguiraud — APPROVED** [review]
LGTM although I wouldn't count posting in "my ROOT app and ideas" as contributing to ROOT

File: contribute/index.md (hunk 1 above)
Comment (Axel-Naumann) [reply contribute/index.md:22]: I believe this is referring to "Guidelines for submitting a contribution" (the mechanism Vassil once introduced)

Comment (Axel-Naumann) [reply contribute/index.md]: What about now?

Comment (eguiraud) [reply contribute/index.md]: :+1:

**Pull request merged**